### PR TITLE
Little translation bugs

### PR DESCRIPTION
**Pull request by juancorr**
There are two little translation bugs in the French astrophysics_fr.properties file: ObjectCount is an integer and the utf8 lambda codification

### CODE REVIEW

File: CA_french/astrophysics_fr.properties [hunk 1]
```diff
@@ -31,7 +31,7 @@ datasetfieldtype.astroFacility.description=L'observatoire ou l'installation o
 datasetfieldtype.astroInstrument.description=L'instrument utilis� pour recueillir les donn�es.
 datasetfieldtype.astroObject.description=Les objets astronomiques repr�sent�s dans les donn�es (Il faut donner, de pr�f�rence, des noms reconnaissables par SIMBAD.)
 datasetfieldtype.resolution.Spatial.description=La r�solution spatiale (angulaire) typique des observations, en degr�s d�cimaux.
-datasetfieldtype.resolution.Spectral.description=La r�solution spectrale typique des observations, indiqu�e sous forme de ratio ?/??.
+datasetfieldtype.resolution.Spectral.description=La r�solution spectrale typique des observations, indiqu�e sous forme de ratio \u03bb/\u0394\u03bb.
```
Comment (mhvezina) [inline CA_french/astrophysics_fr.properties]: Shouldn't this file be in UTF-8? See: https://github.com/scholarsportal/dataverse/pull/30/commits/98e9ac2ab33f533f04e1f515c48fd6b00a26a6f0

File: CA_french/astrophysics_fr.properties [hunk 2]
```diff
@@ -70,7 +70,7 @@ datasetfieldtype.coverage.Temporal.StopTime.watermark=AAAA-MM-JJ
 datasetfieldtype.coverage.Spatial.watermark=
 datasetfieldtype.coverage.Depth.watermark=Entrer un nombre � virgule flottante.
 datasetfieldtype.coverage.ObjectDensity.watermark=Entrer un nombre � virgule flottante.
-datasetfieldtype.coverage.ObjectCount.watermark=Entrer un nombre � virgule flottante.
+datasetfieldtype.coverage.ObjectCount.watermark=Entrer un nombre entier.
```
Comment (mhvezina) [inline CA_french/astrophysics_fr.properties]: Yes indeed, English file says "Enter an integer". Thanks for spotting this error.

File: CA_french/astrophysics_fr.properties (hunk 1 above)
Comment (mhvezina) [reply CA_french/astrophysics_fr.properties]: In fact English version is in UTF-8 : https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs/blob/dataverse-v4.10.1/en_US/astrophysics.properties

**Comment by JayanthyChengan** [comment]
@juancorr - Please note that @mhvezina corrected the french files and it got merged.  Please resolve the conflicts and resubmit the PR.  Thanks.

**Comment by juancorr** [comment]
Hi @JayanthyChengan , I have not many experience with GitHub. Is it all ok now?
Thanks

**Comment by JayanthyChengan** [comment]
@juancorr - please merge the latest code from GlobalDataverseCommunityConsortium:develop branch to your frenchchanges branch.  

It's 3 commits behind GlobalDataverseCommunityConsortium:develop. refer to the image

image

**Comment by juancorr** [comment]
Thanks @JayanthyChengan , I have merged the code.

**Comment by JayanthyChengan** [comment]
@juancorr - that's great and as you notice, the Spanish language files doesn't exists in your branch now.  Please commit them to merge.  Thanks

**Comment by juancorr** [comment]
> @juancorr - that's great and as you notice, the Spanish language files doesn't exists in your branch now. Please commit them to merge. Thanks

@JayanthyChengan  done. Spanish files are now in the frenchChanges branch.

**Comment by juancorr** [comment]
Close. This is not needed with new Spanish and French translations

**Pull request closed without merging**